dockers/docker-sonic-mgmt/Dockerfile.j2

```diff
@@ -250,3 +251,8 @@ RUN python3 -m pip install aiohttp \
 
 # Deactivating a virtualenv
 ENV PATH="$BACKUP_OF_PATH"
+
+USER root
+WORKDIR /azp
+COPY ./start.sh .
+RUN chmod +x start.sh
```

Comment:
After added `start.sh` to docker image, need to set back to default USER and WORKDIR.

Reply:
Accepted. Thanks!